elements/pfe-tabs/src/pfe-tab.scss

```diff
@@ -3,252 +3,379 @@
+    PaddingLeft:             calc(#{pfe-var(container-padding)} * 2),
   )
 );
 
+// TODO: These will be deprecated at 1.0  
+$backwards-compatibility: (
+  focus:                 pfe-local(Color--focus), 
+  highlight:             pfe-local(BorderColor),
+);
+$LOCAL-VARIABLES: map-deep-merge($LOCAL-VARIABLES, $backwards-compatibility);
+
+
 /// ===========================================================================
-/// HORIZONTAL TAB ORIENTATION
+/// DEFAULT / WIND TAB VARIANT
 /// ===========================================================================
 
 :host {
-
+  text-align: left;
   position: relative;
   display: block;
-  margin: #{pfe-local($cssvar: Margin, $region: tab, $fallback: 0 0 -1px)};
-
+  margin: pfe-local(Margin, $region: tab);
   // Padding
-  padding: #{pfe-local(PaddingTop, $region: tab)} #{pfe-local(PaddingRight, $region: tab)} #{pfe-local(PaddingBottom, $region: tab)} #{pfe-local(PaddingLeft, $region: tab)};
-
-  // Border
-  border-top:     #{pfe-local($cssvar: BorderTop, $region: tab, $fallback: #{pfe-local($cssvar: BorderWidth, $region: tab, $fallback: #{pfe-var(ui--border-width)})} #{pfe-var(ui--border-style)} #{pfe-local(BorderColor, transparent, $region: tab)} )};
-  border-right:   #{pfe-local($cssvar: BorderRight, $region: tab, $fallback: #{pfe-local($cssvar: BorderWidth, $region: tab, $fallback: #{pfe-var(ui--border-width)})} #{pfe-var(ui--border-style)} #{pfe-local(BorderColor, transparent, $region: tab)} )};
-  border-bottom:  #{pfe-local(BorderBottom, 0, $region: tab)};
-  border-left:    #{pfe-local($cssvar: BorderLeft, $region: tab, $fallback: #{pfe-local($cssvar: BorderWidth, $region: tab, $fallback: #{pfe-var(ui--border-width)})} #{pfe-var(ui--border-style)} #{pfe-local(BorderColor, transparent, $region: tab)} )};
-
+  padding:
+    pfe-local(PaddingTop, $region: tab) pfe-local(PaddingRight, $region: tab) pfe-local(PaddingBottom, $region: tab) pfe-local(PaddingLeft, $region: tab);
+  // Borders         
+  border-style: pfe-var(ui--border-style);
+  border-width: pfe-var(ui--border-width);
+  border-color: transparent;
+  border-bottom-width: pfe-local(BorderWidth);
   // Inner style
-  background-color: #{pfe-local(BackgroundColor)};
-  color: #{pfe-local(Color)};
-
+  background-color: pfe-local(BackgroundColor);
+  color: pfe-local(Color);
   cursor: pointer;
-  text-align: #{pfe-local(TextAlign, center, $region: tab)};
-  
+  text-align: pfe-local(TextAlign, center, $region: tab);
+
   // Exposing this for customization
-  text-transform: #{pfe-local($cssvar: TextTransform, $region: tab, $fallback: none)};
-  font-family:  #{pfe-var(font-family)};
-  font-weight:  #{pfe-var(font-weight--normal)};
-  font-size:    #{pfe-local(FontSize, $region: tab)};
-    
-  .pfe-tab > * {
-    color: inherit !important;
-    font-size: inherit;
-    font-weight: inherit;
-    margin: 0;
+  text-transform: pfe-local(TextTransform, $region: tab, $fallback: none);
```

Comment:
Should this fallback be listed in the LOCAL-VARIABLES map?

Reply:
moved